compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -4093,6 +4093,47 @@
                     "type": "Token"
                 }
             ]
+        },
+        {
+            "name": "OrderByClauseNode",
+            "base": "ClauseNode",
+            "kind": "ORDER_BY_CLAUSE",
+            "attributes": [
+                {
+                    "name": "orderKeyword",
+                    "type": "Token"
+                },
+                {
+                    "name": "byKeyword",
+                    "type": "Token"
+                },
+                {
+                    "name": "orderKey",
+                    "type": "OrderKeyNode",
+                    "occurrences": "MULTIPLE_SEPARATED"
+                }
+            ]
+        },
+        {
+            "name": "OrderKeyNode",
+            "base": "Node",
+            "kind": "ORDER_KEY",
+            "attributes": [
+                {
+                    "name": "expression",
+                    "type": "ExpressionNode"
+                },
+                {
+                    "name": "ascendingKeyword",
+                    "type": "Token",
+                    "isOptional": true
+                },
+                {
+                    "name": "descendingKeyword",
+                    "type": "Token",
+                    "isOptional": true
+                }
```

Comment:
Defining both as optional allows someone to add both keywords at once in the syntax-tree. 
I think we need to have only one optional-field and that should hold the value regardless its ascending or descending.